### DISCUSSION TITLE
24.10.12 - Finds "undiscoverable devices" and there is no backup option?

### DISCUSSION THREAD

**Discussion by RompitaKablo** [netalertx/NetAlertX#846] [Q&A] [answered]
Prior to upgrading to v24.10.12 I had created 3 undiscoverable devices. These were all dumb, unmanaged switches, so I could create a better network map. Since v24.10.12 these 3 switches now appear a second time but as new devices found by NetAlertX and they each have a mac address (all not vendor issued). I did not see this mentioned in the release notes. What is NetalertX doing to create these records?

Also, since v24.10.12, there is no option to backup / restore, just create a CSV. Has backup been deprecated?

![image](https://github.com/user-attachments/assets/6af8d961-d8e8-4892-ac96-e3fb24bed829)

**Comment by jokob-sk** [comment, marked as answer]
Hi @RompitaKablo ,

Thanks for the questions! Let me try to explain.

## 1

> 3 switches now appear a second time but as new devices found by NetAlertX and they each have a mac address

This is because the MAC generation wasn't working for unrecoverable devices (that was a bug) so they will re-appear in the list. You can delete the old ones and use the newly generated ones. This was mentioned in the previous release notes: https://github.com/jokob-sk/NetAlertX/releases/tag/v24.9.26

The MAC is auto-generated based on the device name. 

## 2

Yes, the DB backup option was removed, as mentioned in the UI section of the release notes: https://github.com/jokob-sk/NetAlertX/releases/tag/v24.10.12 (let me know your feedback, but trying to simplify the code where possible and I didn't see anyone using this as in docker you backup volumes/mount points).

**Reply by RompitaKablo** [reply]
Thank you for your time and patience. Obviously I will have to pay better attention to the release notes and become more familiar with this excellent software.

**EDIT:** The point about using Docker for the backup makes sense.

**Reply by jokob-sk** [reply]
Haha, not sure if it is excellent, but thank you for the kind words :) 

All good, glad it makes sense. 

Have a great day,
j
